src/js/pages/InventoryListPage.js

```diff
@@ -197,11 +202,16 @@ export default class InventoryListPage extends Component {
           <Image source={require('../../img/peek.png')} style={styles.peek_img} />
           <View style={styles.secondary_header}>
             <Text style={styles.header_title}>Products</Text>
-            <ModalSelector data={sortOptions} initValue="Name (A to Z)"
+            <ModalSelector data={sortOptions} initValue={i18n.t('modalSelector.azLabel')}
                            style={styles.item_sort} selectTextStyle={styles.sort_text}
-                           onChange={(sortOption) => this.changeSort(sortOption.key)} />
+                           onChange={(sortOption) => this.changeSort(sortOption.key)}
+                           cancelText={i18n.t('modalSelector.cancel')}
+                           listItemAccessible
+                           cancelButtonAccessible
+                           openButtonContainerAccessible
```

Comment:
should these attributes have values assigned, e.g. `listItemAccessible={true}`?

Reply:
Nope, the name will by default be translated to `true`, only if the value needs to be different we need to add it.